spec/controllers/line_items_controller_spec.rb

```diff
@@ -32,9 +32,13 @@
       order = create(:completed_order_with_totals)
       item = create(:line_item, order: order)
       while !order.completed? do break unless order.next! end
+      order.reload
```

Comment:
Huh, I just tested it and it looks like it isn't needed...maybe I was using a different factory before. I'll push another commit, hang on.